frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+import { FilterSet, FilterValue, RangeFilterValue } from "../../models/paged_items";
+
+interface FiltersInfosProps {
+    filters: FilterSet
+}
+
+const FiltersInfos = ({ filters }: FiltersInfosProps) => {
+    const appliedFilters = Object.keys(filters).filter((key: any) => filters[key]?.value)
```

Comment:
There is no need for this `any` typing on `key` - the compiler knows what type to expect from Object.keys